src/dag/meta_generated.rs

```diff
@@ -0,0 +1,118 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Only tangentially related to this, I've been debating introducing a `Makefile` in the root to make the various lint and testing commands I'm using easier to access, and that would also be a good place to store whatever invocation creates this.

Are you open to that, opposed, or would you want more of an argument before deciding?

Reply:
In favor of the idea, but never took the time to learn Makefile syntax. Can we use bash instead? (I think actually we should probably bazelfy this at some point, but...)